src/test/java/codeu/controller/RocketBotHandlerTest.java

```diff
@@ -0,0 +1,72 @@
+package codeu.controller;
+
+import codeu.model.data.BotRequest;
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+import codeu.model.store.basic.MessageStore;
+import codeu.model.store.persistence.PersistentStorageAgent;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import org.junit.Assert;
+
+public class RocketBotHandlerTest {
+  private UserStore mockUserStore;
+  private MessageStore mockMessageStore;
+  private final UUID CONVERSATION_ID_ONE = UUID.randomUUID();
+  private final UUID USER_ID_ONE = UUID.randomUUID();
+  private RocketBotHandler rocketBot;
+
+  @Before
+  public void setup() {
+    PersistentStorageAgent mockPersistentStorageAgent = Mockito.mock(PersistentStorageAgent.class);
+    mockUserStore = UserStore.getTestInstance(mockPersistentStorageAgent);
+    mockMessageStore = MessageStore.getTestInstance(mockPersistentStorageAgent);
+    rocketBot = new RocketBotHandler();
+    rocketBot.setUserStore(mockUserStore);
+    rocketBot.setMessageStore(mockMessageStore);
+  }
+
+  /**
```

Comment:
These tests should probably test the user store method itself directly and you should add a test that checks the message was written